index.js

```diff
@@ -312,16 +316,16 @@ FeatureService.prototype.featureCount = function (callback) {
  * @param {number} pages - the number of pages we'll create
  * @param {number} max - the max number of feature per page
```

Comment:
noticing jsdoc is pretty inaccurate in other parts not changed by this PR in index.js